src/HttpClient/ForbiddingHttpClient.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace eLife\ApiClient\HttpClient;
+
+use eLife\ApiClient\HttpClient;
+use GuzzleHttp\Promise\PromiseInterface;
+use LogicException;
+use Psr\Http\Message\RequestInterface;
+use function GuzzleHttp\Psr7\str;
+
+final class ForbiddingHttpClient implements HttpClient
+{
+    public function send(RequestInterface $request) : PromiseInterface
+    {
+        throw new LogicException('Forbidden call: '.str($request));
```

Comment:
Not sure about the intended usage. If it's for constraining code not to use HTTP calls, the two mentioned exceptions describe an error happened while making a request, not a request that should not have been made. Would be happy to create `UnintendedRequest` or `UnintendedInteraction` which `extends `ApiException`.